omnigan/blocks.py

```diff
@@ -372,27 +371,24 @@ def __init__(
         spade_use_spectral_norm,
         spade_param_free_norm,
         spade_kernel_size,
+        fullspade=True,
     ):
         """Create a SPADE-based decoder, which forwards z and the conditioning
         tensors seg (in the original paper, conditioning is on a semantic map only).
-
         All along, z is conditioned on seg. First 3 SpadeResblocks (SRB) do not shrink
         the channel dimension, and an upsampling is applied after each. Therefore
         2 upsamplings at this point. Then, for each remaining upsamplings
         (w.r.t. spade_n_up), the SRB shrinks channels by 2. Before final conv to get 3
         channels, the number of channels is therefore:
-
             final_nc = channels(z) * 2 ** (spade_n_up - 2)
-
-
         Args:
-            latent_shape (tuple): z's shape (only the number of channels matters)
+            latent_dim (tuple): z's shape (only the number of channels matters)
             cond_nc (int): conditioning tensor's expected number of channels
             spade_n_up (int): Number of total upsamplings from z
             spade_use_spectral_norm (bool): use spectral normalization?
             spade_param_free_norm (str): norm to use before SPADE de-normalization
             spade_kernel_size (int): SPADE conv layers' kernel size
-
+            fullspade (bool): Produces an image as a final  result
```

Comment:
Where is this parameter used?

Reply:
It's not. Good catch, I'll get rid of it